tests/ICU4N.Tests/Dev/Test/Lang/UCharacterSurrogateTest.cs

```diff
@@ -412,7 +424,13 @@ internal void Fail(String text, int index, int offset, Type exc)
                 catch (Exception e)
                 {
                     //if (!exc.isInstance(e))
-                    if (!exc.GetTypeInfo().IsAssignableFrom(e.GetType()))
+                    bool isAssignableForm =
+#if FEATURE_TYPEEXTENSIONS_GETTYPEINFO
+                    exc.GetTypeInfo().IsAssignableFrom(e.GetType());
```

Comment:
Please indent this 4 spaces so it is clear it is a line continuation, not a new line.

Reply:
Okay, i will fix this in the next commit.